functions/src/definitions/eventHandlers/onMessageUpdate.ts

```diff
@@ -42,7 +42,27 @@ const onMessageUpdateV2 = onDocumentUpdated(
         rationalisation: rationalisation,
       })
       await replyPendingInstances(postChangeSnap)
+      if (
+        messageData?.communityNote?.downvoted &&
+        messageData?.communityNote?.pendingCorrection
+      ) {
+        await replyCommunityNoteInstances(postChangeSnap)
+      }
+    }
+
+    if (
```

Comment:
Only reply community note instances under 2 circumstances

1. downvoted tag switched to true and messageData is assessed
2. isAssessed tag switched to true and pendingCorrection tag (new thing I added) is true
3. Also means there's a case where we need to switch pendingCorrection to true if downvoted tag switched to true but messageData not yet assessed